Problem building latest develop in OS X

Is `size_t` in OS X defined differently than Linux? My guess is that it works in Linux because it is defined as `unsigned int` but jsoncpp does not have a constructor for `unsigned long` so it fails in OS X.

```
./libsolidity/ast/ASTJsonConverter.cpp:45:15: error: conversion from 'size_t' (aka 'unsigned long') to 'Json::Value' is ambiguous
        node["id"] = _node.id();
                     ^~~~~~~~~~
/usr/local/include/json/value.h:285:3: note: candidate constructor
  Value(Int value);
  ^
/usr/local/include/json/value.h:286:3: note: candidate constructor
  Value(UInt value);
  ^
/usr/local/include/json/value.h:288:3: note: candidate constructor
  Value(Int64 value);
  ^
/usr/local/include/json/value.h:289:3: note: candidate constructor
  Value(UInt64 value);
  ^
/usr/local/include/json/value.h:291:3: note: candidate constructor
  Value(double value);
  ^
/usr/local/include/json/value.h:314:3: note: candidate constructor
  Value(bool value);
  ^
/usr/local/include/json/value.h:325:26: note: passing argument to parameter 'other' here
  Value& operator=(Value other);
                         ^
./libsolidity/ast/ASTJsonConverter.cpp:127:34: error: conversion from 'size_t' (aka 'unsigned long') to 'const Json::Value' is ambiguous
                linearizedBaseContracts.append(baseContract->id());
                                               ^~~~~~~~~~~~~~~~~~
/usr/local/include/json/value.h:285:3: note: candidate constructor
  Value(Int value);
  ^
/usr/local/include/json/value.h:286:3: note: candidate constructor
  Value(UInt value);
  ^
/usr/local/include/json/value.h:288:3: note: candidate constructor
  Value(Int64 value);
  ^
/usr/local/include/json/value.h:289:3: note: candidate constructor
  Value(UInt64 value);
  ^
/usr/local/include/json/value.h:291:3: note: candidate constructor
  Value(double value);
  ^
/usr/local/include/json/value.h:314:3: note: candidate constructor
  Value(bool value);
  ^
2 errors generated.
make[2]: *** [libsolidity/CMakeFiles/solidity.dir/ast/ASTJsonConverter.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [libsolidity/CMakeFiles/solidity.dir/all] Error 2
make: *** [all] Error 2
```

--- a/libsolidity/ast/ASTJsonConverter.cpp
+++ b/libsolidity/ast/ASTJsonConverter.cpp
@@ -42,7 +42,7 @@ void ASTJsonConverter::addJsonNode(
 {
 	Json::Value node;
 
-	node["id"] = _node.id();
+	node["id"] = Json::UInt64(_node.id());
 	node["src"] = sourceLocationToString(_node.location());
 	node["name"] = _nodeName;
 	if (_attributes.size() != 0)
@@ -124,7 +124,7 @@ bool ASTJsonConverter::visit(ContractDefinition const& _node)
 {
 	Json::Value linearizedBaseContracts(Json::arrayValue);
 	for (auto const& baseContract: _node.annotation().linearizedBaseContracts)
-		linearizedBaseContracts.append(baseContract->id());
+		linearizedBaseContracts.append(Json::UInt64(baseContract->id()));
 	addJsonNode(_node, "ContractDefinition", {
 		make_pair("name", _node.name()),
 		make_pair("isLibrary", _node.isLibrary()),

